exampleExternalDependencies/mongo/docker-compose.yml

```diff
@@ -10,4 +11,3 @@ services:
       - 27017:27017
     volumes:
       - ./init-mongo.js:/docker-entrypoint-initdb.d/init-mongo.js:ro
-      - ./mongo-volume:/data/db
```

Comment:
Please create another docker-compose file with the custom name without the volume.
It would be useful to have both of them. Many people turn down their containers once they stop their work and come back to them once they start again. Losing all the data doesn't sound like a good idea.